### PR TITLE
LUN-465: Remove react native snap carousel

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

**Review by LeandraH — APPROVED** [review]
Nice, tested on iOS emulator

File: src/components/ImageCarousel.tsx [hunk 1]
```diff
@@ -60,13 +76,10 @@ const ImageCarousel = ({ images, minimized = false }: ImageCarouselProps): React
...
-        <Pagination
-          activeDotIndex={currentIndex - 1}
-          dotsLength={allSize}
-          dotStyle={{ backgroundColor: theme.colors.primary }}
-        />
-      )}
+      {!!allSize &&
+        !!currentIndex &&
+        allSize > 1 &&
+        imagesUrls.map((item, index) => (index + 1 === currentIndex ? <ActiveDot /> : <Dot />))}
```
Comment (f1sh1918) [inline src/components/ImageCarousel.tsx:82]:
```suggestion
        imagesUrls.map((item, index) => (index + 1 === currentIndex ? <ActiveDot key={item.url}/> : <Dot key={item.url}/>))}
```

Comment (f1sh1918) [reply src/components/ImageCarousel.tsx:82]: missing keys

**Review by f1sh1918 — APPROVED** [review]
nice refactoring. Just keys are missing.
I wonder if the design has some ideas to display the indicators a bit smarter. I think they are getting lost

**Comment by SteffiStuffel** [comment]
> I wonder if the design has some ideas to display the indicators a bit smarter. I think they are getting lost

Sure there are better options, but I think this is out of scope. Currently we have way more important topics, where we can improve lunes, so i would just keep it, open no tickets or so for now. But if you think it is important now, feel free to open a design ticket.

**Pull request merged**